crates/block-producer/src/withdrawal.rs

```diff
@@ -303,6 +321,20 @@ pub fn unlock_to_owner(
     }))
 }
 
+fn is_v1_withdrawal_cell(withdrawal_cell: &CellInfo) -> bool {
```

Comment:
```suggestion
fn is_block_finalized_withdrawal_cell(withdrawal_cell: &CellInfo) -> bool {
```

Reply:
"is_block_finalized_withdrawal_cell" is more confusing for me. How about "is_legacy_withdrawal_cell"?